.github/workflows/unit_tests.yml

```diff
@@ -212,7 +212,7 @@ jobs:
           # run test suite
           python -O -m test.framework.suite 2>&1 | tee test_framework_suite.log
           # try and make sure output of running tests is clean (no printed messages/warnings)
-          IGNORE_PATTERNS="no GitHub token available|skipping SvnRepository test|requires Lmod as modules tool|stty: 'standard input': Inappropriate ioctl for device|CryptographyDeprecationWarning: Python 3.[56]|from cryptography.*default_backend|CryptographyDeprecationWarning: Python 2|from cryptography.utils import int_from_bytes|Blowfish|GC3Pie not available, skipping test"
+          IGNORE_PATTERNS="no GitHub token available|skipping SvnRepository test|requires Lmod as modules tool|stty: 'standard input': Inappropriate ioctl for device|CryptographyDeprecationWarning: Python 3.[56]|from cryptography.*default_backend|CryptographyDeprecationWarning: Python 2|from cryptography.utils import int_from_bytes|Blowfish|GC3Pie not available, skipping test|\.s+"
```

Comment:
Adding `\s+` to `IGNORE_PATTERNS` defeats the whole purpose of it?

Reply:
Pythons unittest will print a literal dot for each test executed and a literal s for each test skipped.

I did add `\.s+` not `\s+` to the patterns, i.e. I want to ignore a dot followed by one or more "s" as otherwise those skipped tests would make CI fail. See my last paragraph in the PR description about a discussion starter on this.

Reply:
Oh, right, I totally misinterpreting this, sorry... Makes sense